manifests/r/Renode/Renode/1.12.0/Renode.Renode.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Renode.Renode
+PackageVersion: 1.12.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```

```suggestion
  Architecture: x64
```
Try it with `x64`.

Revo Uninstaller shows it's `x64` and it places the files in `C:\Program Files\Renode`.